src/include/access/k2/pg_ddl.h

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
do we need this file?

Reply:
will put all ddl related logic into this file and pass the calls to catalog manager. Should be implemented by Ahsan.